static/js/zamboni/devhub.js

```diff
@@ -338,6 +338,7 @@ function addonFormSubmit() {
         };
         $('.edit-media-button button').prop('disabled', false);
         $('form', parent_div).submit(function(e){
+            console.log('AAAAAAAAAAAAAAAA', e);
```

Comment:
Yeah, maybe not :)

Reply:
Eh, damn, I already removed that but in a different branch. Too many branches :see_no_evil: 

Fixed in d060f239d7